src/main/kotlin/org/rust/cargo/project/settings/RsExternalLinterProjectSettingsService.kt

```diff
@@ -0,0 +1,65 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.cargo.project.settings
+
+import com.intellij.openapi.components.State
+import com.intellij.openapi.components.Storage
+import com.intellij.openapi.components.StoragePathMacros
+import com.intellij.openapi.project.Project
+import org.rust.cargo.project.settings.RsExternalLinterProjectSettingsService.RsExternalLinterProjectSettings
+import org.rust.cargo.toolchain.ExternalLinter
+
+val Project.externalLinterSettings: RsExternalLinterProjectSettingsService
+    get() = getService(RsExternalLinterProjectSettingsService::class.java)
+        ?: error("Failed to get ${SERVICE_NAME}Service for $this")
```

Comment:
Why not just `service()`?
It also prints the similar error

The same question is for all other similar extensions